### PR TITLE
add one more test note

### PR DESCRIPTION
**Pull request by OlgaDidyk**
Please check my test note

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-some test note!
\ No newline at end of file
+some test note!
```
Comment (sankouski-dzmitry) [inline test.txt]: Please, add more notes

Comment (OlgaDidyk) [reply test.txt]: i have added more notes. You can look at them tomorrow) it is too late now

Comment (OlgaDidyk) [reply test.txt]: i have added more notes. You can look at them tomorrow) it is too late now

File: test.txt [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
-one more test note on the test branch
\ No newline at end of file
+one more test note on the test branch
+
+
+More notes:
+
+Contributions to a source code repository that uses a distributed version control system are commonly made by means of a pull request, also known as a merge request.[9] The contributor requests that the project maintainer pull the source code change, hence the name "pull request". The maintainer has to merge the pull request if the contribution should become part of the source base.[10]
+
+The developer creates a pull request to notify maintainers of a new change; a comment thread is associated with each pull request. This allows for focused discussion of code changes. Submitted pull requests are visible to anyone with repository access. A pull request can be accepted or rejected by maintainers.[11]
```
Comment (sankouski-dzmitry) [inline test.txt]: my comment

**Pull request merged**